care/facility/api/viewsets/daily_round.py

```diff
@@ -45,22 +45,20 @@ def get_serializer(self, *args, **kwargs):
             ).id
         return super().get_serializer(*args, **kwargs)
 
-    @action(methods=["POST"], detail=False)
```

Comment:
This endpoint should only be performed over POST to ensure that the filters do not exceed the max allowed character limit in GET, change the permission instead of the HTTP method, which means no changes in FE as well.